.github/workflows/pr-markdown.yml

```diff
@@ -22,17 +18,14 @@ jobs:
           application_id: ${{ secrets.ADVANCED_SECURITY_APP_ID }}
           application_private_key: ${{ secrets.ADVANCED_SECURITY_APP_KEY }}
 
-      - name: Install Deps
-        run: python3 -m pip install pipenv && python3 -m pipenv install
+      - name: Generate Markdown
+        uses: advanced-security/secret-scanning-tools@fix-action
```

Comment:
We should either using a pinned commit, released version, or `main`.

Reply:
Yep, that was to test it against my fix before it was merged! Good spot.

I'll cut a `v1` release on `secret-scanning-tools`, which I didn't do before since I hadn't confirmed it worked on this repo - now it does, I'm ready to cut a release.

When I've done that, I'll target this at `v1` (rather than a pinned commit, it's also owned by us so there's no advantage in pinning to a commit)